### PR TITLE
Delete class files and add *.class in .gitignore

### LINKED ISSUE #1: Delete object file
Delete the class file in bin directory.

### PR DESCRIPTION
**Pull request by dareunk**
1. Delete the class file already existed in bin directory.
2. Add *.class in .gitignore to exclude *.out from git ever.

resolved: #1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.class
```
Comment (happyeun12) [inline .gitignore:1]: You should put some exception below in .gitignore
```suggestion
*.o
*.out
*.exe 
```

**Review by happyeun12 — CHANGES_REQUESTED** [review]
Well done. Thanks for helping our open source to develop. Check the review

File: .gitignore (hunk 1 above)
Comment (dareunk) [reply .gitignore:1]: Thanks for your feedback. I'll work on it

**Review by happyeun12 — APPROVED** [review]
Well done

**Pull request merged**